tinyagentos/routes/receipts.py

```
@@ -0,0 +1,100 @@
+    files_changed: list | None = None
+    stop_reason: str = ""
+    redactions: list | None = None
+    human_approval: dict | None = None
+    trace_id: str = ""
+    board_audit_event_id: str = ""
+    decision_id: str = ""
+    metadata: dict | None = None
+
+
+@router.post("/api/receipts")
+async def create_receipt(body: ReceiptIn, request: Request, user: CurrentUser = Depends(current_user)):
```

Ruff (0.15.20)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)